### ISSUE TITLE
invalid token, invalid signature, and rust failed to load

### ISSUE THREAD

**Issue by DBDigital** [simonrob/email-oauth2-proxy#294]
I got email-oauth2-proxy to install on Python 3.8 (windows version and no errors on install other than suggesting I add the scripts folder to path which I did)

I downloaded and configured the emailproxy.config file, and stuck it into C:\Users\user1\AppData\Local\Programs\Python\Python38\Scripts

But when I do 
py -m emailproxy 
command I get:

line 55 in module from cryptography.fernet import fernet, multifernet, invaidtoken

line 14 from cryptography.exceptions import invalid signature

line 9 from cryptography.hazmat.bindings._rust import exceptions as rust_exceptions ImportError: DLL file failed while importing _rust: the specified procedure could not be found

**Comment by simonrob** [comment]
I'm afraid this is another issue related to your older operating system, rather than the proxy. [This search result](https://stackoverflow.com/questions/78853596/how-to-solve-importerror-dll-load-failed-while-importing-rust) may help.

**Comment by DBDigital** [comment]
Ah drat, that link is helpful, thank you.  Although I am very new to Python, I will try to uninstall cryptography (to version 41.0.7) and bcrypt (to version 3.2.0) packages.

**Comment by DBDigital** [comment]
Good news I got it going!  I did as suggested:
py -m pip uninstall cryptography
py -m pip uninstall bcrypt

then got
https://files.pythonhosted.org/packages/f6/23/b28f4a03650512efff13a8fcbb977bac178a765c5a887a6720bee13fa85b/cryptography-41.0.7-cp37-abi3-win_amd64.whl

https://files.pythonhosted.org/packages/21/8d/ed20081491e71f078e61804fe0c8250167008cf3ff594e1fb396cf138f2b/bcrypt-3.2.0-cp36-abi3-win_amd64.whl

And installed those with:
py -m pip install C:\yourdirectory\cryptography-41.0.7-cp37-abi3-win_amd64.whl

py -m pip install C:\yourdirectory\bcrypt-3.2.0-cp36-abi3-win_amd64.whl

I had tried installing from PyPi with commands such as py -m pip cryptography==41.0.7 but never worked.

Anyway that installed and it then ran with
py -m emailproxy
No problems at start.  Connecting from thunderbird I did have a little problem until I used the authorize/external authorization mode.  Then it worked.

Well after I updated to client ID 9e5f94bc-e8a4-4e73-b8be-63364c29d753 with no secret.

Whew.  Thanks for a great piece of software!
